docs/src/main/sphinx/connector/postgresql.rst

```diff
@@ -93,6 +93,34 @@ catalog named ``sales`` using the configured connector.
 Type mapping
 ------------
 
+The data type mappings are as follows:
+
+=================== ================================ =======================================================================
+PostgreSQL          Trino                            Notes
+=================== ================================ =======================================================================
+``BIT``             ``BOOLEAN``
+``SMALLINT``        ``SMALLINT``
+``INTEGER``         ``INTEGER``
+``BIGINT``          ``BIGINT``
```

Comment:
This reminds that we should probably also test smallserial, serial and bigserial types too.

Reply:
https://github.com/trinodb/trino/pull/11385